OSVRUnreal/Plugins/OSVR/Source/OSVR/Private/OSVRHMD.cpp

```diff
@@ -170,9 +172,10 @@ bool FOSVRHMD::IsChromaAbCorrectionEnabled() const
 	return false;
 }
 
-ISceneViewExtension* FOSVRHMD::GetViewExtension()
+TSharedPtr<class ISceneViewExtension, ESPMode::ThreadSafe> FOSVRHMD::GetViewExtension()

```

Comment:
The logic here is questionable - copy-constructing for a "get view extension" method that feels more like a built-in downcast (that is, that sounds like it should return a more specific interface). Will have to investigate this more.